### DISCUSSION TITLE
Enable browsing by folder for file/image selection from file library.

### DISCUSSION THREAD

**Discussion by neilcamilleri** [directus/directus#3464] [Feature Requests]
When opting to use a file from the file library for the file/image field, in the right side drawer that opens up, it would be great to be able to browse by folder. Especially after you've sorted your file collection into folders in the first place 😅 

Currently, you can only sort by folders but there isn't even an indication which folder you're browsing. In instances with hundreds of different files uploaded it can be hard to browse for the right one.

**Comment by benhaynes** [comment]
I think a very important piece to this (which should solve this everywhere, without showing/adding sidebars) is to show all contained folders at the top of the file listing. Below is a design... apologies for it being a bit dated in our design system.

![Item Browse Grouping](https://user-images.githubusercontent.com/522079/131744602-b94daf8d-9920-440e-9140-a8f06e699da6.png)

**Comment by rijkvanzanten** [comment]
This feature request has been accepted and is queued to be implemented! You can follow along with the progress here: https://github.com/directus/directus/issues/15968
